### ISSUE TITLE
Small ui bug

### ISSUE THREAD

**Issue by binderclip** [HappenApps/Quiver#184]
![small-ui-bug](https://cloud.githubusercontent.com/assets/8187479/9954955/b6d32c44-5e1f-11e5-87dc-f9dca4c70759.png)
![small-ui-bug2](https://cloud.githubusercontent.com/assets/8187479/9954954/b6d22a38-5e1f-11e5-86da-756f1ae0739f.png)

There are some white block on the scrollbar.

**Comment by ylian** [comment]
Wow. Good eye. It took me a few minutes to see those little white blocks even with your instructions. No idea what's causing it yet.

**Comment by ylian** [comment]
It's a Yosemite bug (http://stackoverflow.com/questions/28383563/cocoa-backgrounds-of-components-on-nsvisualeffectview-rendering-incorrectly-whe). Basically the scrollbar, the count label and the translucent sidebar don't go well together.

Apparently this has been fixed on El Capitan. I just took this screenshot on El Capitan:

![screen shot 2015-09-18 at 8 25 31 pm](https://cloud.githubusercontent.com/assets/984895/9973641/c94047fc-5e44-11e5-9504-51f00c50c890.png)

I won't bother fixing this since it's minor and El Capitan is coming out by the end of month.
